apis/config/v1beta1/configuration_types.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The Kubernetes Authors.
+Copyright 2023 The Kubernetes Authors.
```

Comment:
revert

The copyright year should be the year when the file was first added.

Reply:
Makes sense. Reverted.